app/models/notifications/queries/MucrInfo.scala

```diff
@@ -25,7 +25,11 @@ case class MucrInfo(
   entryStatus: Option[EntryStatus] = None,
   isShut: Option[Boolean] = None,
   movements: Seq[MovementInfo] = Seq.empty
-)
+) extends UcrInfo {
+  def transport: Option[Transport] =
+    movements.find(_.transportDetails.isDefined).flatMap(_.transportDetails)
```

Comment:
This could be moved to `UcrInfo`

Reply:
Good idea.  The common trait didn't exist when I wrote this.   Keep the suggestions coming - I'll create a new PR